contracts/PolyMathTokenOffering.sol

```diff
@@ -118,21 +124,38 @@ contract PolyMathTokenOffering is Ownable {
     }
    }
 
+   function ethToTokens(uint256 ethAmount) internal returns (uint256) {
+    uint256 tokenAmount;
+    uint256 bonusRate = calculateBonusRate();
+    tokenAmount = (ethAmount.mul(bonusRate)).div(1 ether);
+    return tokenAmount;
+   }
+
   // low level token purchase function
+  event Loggyboy(string m);
```

Comment:
what a weird name

Reply:
Not a fan of the loggy boy? :joy:

Reply:
It was accidentally left in, it's removed in an upcoming PR ;)